tests/new_integration_tests/test_config_category.py

```diff
@@ -139,7 +164,6 @@ def test_end_message_respects_config_file_argument(
             )
 
         # assert
-        assert (
-            f"Configuration file written to {config_file}"
-            in capsys.readouterr().out
+        assert capsys.readouterr().out.endswith(
+            f"Configuration file written to {config_file}\n"
```

Comment:
This change is unrelated to your patch. Even though it looks fine, I'm hesitant to merge something that changes production code _and_ a seemingly unrelated test.

Would you mind reverting this change?

Reply:
Done.

I can open a new PR with just this change?

Reply:
Sure!